integration-tests/deployment/ccip/state.go

```diff
@@ -290,6 +292,15 @@ func LoadChainState(chain deployment.Chain, addresses map[string]deployment.Type
 				return state, err
 			}
 			state.Receiver = mr
+		case deployment.NewTypeAndVersion(PriceFeed, deployment.Version1_0_0).String():
+			feed, err := aggregator_v3_interface.NewAggregatorV3Interface(common.HexToAddress(address), chain.Client)
+			if err != nil {
+				return state, err
+			}
+			if state.Feeds == nil {
+				state.Feeds = make(map[string]*aggregator_v3_interface.AggregatorV3Interface)
+			}
+			state.Feeds[tvStr.String()] = feed
```

Comment:
I don't think tvStr will be unique here, they'll all be of the type OffchainAggregator (https://github.com/smartcontractkit/libocr/blob/1a7e0789ba965e9a1cdfe6fb4d5e31acd1627025/contract/OffchainAggregator.sol#L112) or something. Maybe best to key them on `description()` e.g. "LINK / USD" see an example here of a real feed https://sepolia.arbiscan.io/address/0x0FB99723Aee6f420beAD13e6bBB79b7E6F034298#readContract

Reply:
I like the idea of using description 👍. I was going to prefix it with the token name but description is cleaner in this case.